templates/Angular2Spa/webpack.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = (env) => {
             rules: [
                 { test: /\.ts$/, include: /ClientApp/, use: ['awesome-typescript-loader?silent=true', 'angular2-template-loader'] },
                 { test: /\.html$/, use: 'html-loader?minimize=false' },
-                { test: /\.css$/, use: ['to-string-loader', 'css-loader'] },
+                { test: /\.css$/, use: [ 'to-string-loader', isDevBuild ? 'css-loader' : 'css-loader?minimize' ] },
```

Comment:
In a few cases you've added spaces inside the `[...]` array, such as in this file, but not in some other cases (e.g., for ReactSpa's `webpack.config.js`).

Would you be able to make the changes consistent with each other? Personally I'd prefer not having the extra spaces in any of them, but if you prefer them to be there, that's OK as long as they are all the same :) Thanks!